lua/nvim_lsp/metals.lua

```diff
@@ -27,17 +27,23 @@ local function make_installer()
       error('Need "JDK" to install this.')
       return
     end
-    local coursier_exe
-    if not (util.has_bins("cs")) then
+
+    local coursier_exe = nil
+    if util.has_bins("coursier") then
```

Comment:
I think we should probably reverse this check to look for `cs` first. I a lot of scenarios, if the user has `coursier`, they will have `cs`, and it's more preferable to use the native `cs`. I'd rather use the `coursier` exec as a backup. Is there a reason you have them in this order?

Reply:
ok, I've changed the order now